frontend/src/router/routeHandlers.ts

```diff
@@ -84,7 +84,7 @@ const beforeEnterGrantUserPermissionRoute = async (
         return { path: routeItems.dashboard.path };
     }
 
-    populateBreadcrumb([routeItems.dashboard, routeItems.grantUserPermission]);
```

Comment:
Remove previous "last" (same page) item from breadcrumb list.

Reply:
This isn’t directly related to your PR, just an observation. 
Handling breadcrumbs at the route level is interesting, but it might be more flexible to manage them at the view level. I think it offers greater flexibility and is easier to implement, though doing it at the route level does enforce hierarchy well.

Reply:
Thanks for the point!! I didn't know the past communication for this breadcrumb's design from the frontend developers. But I feel, same as you, I would favor that to be in the components with easy configuration to use, the route does not need to care too much handling it I feel.